tests/utils.py

```diff
@@ -42,6 +57,13 @@ def load_script(source_file_folder, source_file_name):
         return f.read()
 
 
+def load_datacube_code():
```

Comment:
Is this needed already? I think it's not called anywhere.

Reply:
No it is not needed, made a mistake when I was copying another part of code. Will delete it.

Reply:
Thanks :+1: